### DISCUSSION TITLE
GitHub page的使用简化

### DISCUSSION THREAD

**Discussion by mks10001** [community/community#133362] [New to GitHub]
GitHub page与gridea的远程连接总是失败？能否有更好的方式连接写作客户端

**Comment by SvenSchneiderDVAG** [comment]
I asked AI what the TO asked here. It's chinese and means: 
> "Remote connection to Gridea always fails. Is there a better way to connect to the writing client?"

**Comment by ghostinhershell** [comment]
Hi @mks10001, thanks for participating!

We take our [Code of Conduct](https://github.com/community/community/blob/920a928ffe3ae4863b64a4ea26917046e11f5c1a/CODE_OF_CONDUCT.md) very seriously and want to help ensure that everyone has a good experience free of antagonism and harassment. Unfortunately, we don’t currently have moderators for languages other than English. Until that changes, we need to ask that everyone use English here in the GitHub Community when posting. We’ll be locking any posts in languages other than English for now, including this one.
